docs/administration-guide/10-single-sign-on.md

```diff
@@ -82,20 +97,44 @@ As you can see below, if you have an **Accounting** group in both your LDAP serv
 - Updates to a person's group membership based on LDAP mappings are not instantaneous; the changes will take effect only after people log back in.
 - People are only ever added to or removed from mapped groups; the sync has no effect on groups in your Metabase that don't have an LDAP mapping.
 
-## LDAP advanced features
+### LDAP group membership filter
 
 {% include plans-blockquote.html feature="LDAP advanced features" %}
 
-### LDAP syncing user attributes
-
-If you're running a [paid version of Metabase](https://www.metabase.com/pricing) and using [data sandboxes](../enterprise-guide/data-sandboxes.md), you can use existing LDAP [user attributes](../enterprise-guide/data-sandboxes.html#getting-user-attributes) when granting sandboxed access.
-
-### LDAP group membership filter
-
 Group membership lookup filter. The placeholders {dn} and {uid} will be replaced by the user's Distinguished Name and UID, respectively.
 
 ## Further reading
 
 - [Using LDAP for authentication and access control](/learn/permissions/ldap-auth-access-control.html)
 - [LDAP troubleshooting guide](../troubleshooting-guide/ldap.md)
 - [Permissions overview](05-setting-permissions.md)
+
+## Syncing user attributes at login
+
+{% include plans-blockquote.html feature="Advanced authentication features" %}
+
+### Syncing attributes with LDAP
+
+LDAP user attributes are managed from your LDAP directory. If you've set up [data-sandboxing-docs][data-sandboxing-docs], these attributes will be [automatically passed to Metabase][getting-user-attributes].
```

Comment:
Can you rephrase to active voice so it's clear what's managing and passing attributes?